report_thesis/src/sections/background.tex

```diff
@@ -206,7 +206,72 @@ \subsubsection{Support Vector Regression (SVR)}
 
 \subsubsection{Extra Trees Regressor (ETR)}
 
-\subsubsection{Gradient Boosting Regressor (GBR)}
+\subsubsection{Gradient Boosting Regression (GBR)}\label{sec:gradientboost}
+In this section we introduce \gls{gbr} primarily based on \citet{James2023AnIS}.
+Gradient boosting builds on simpler concepts and incorporates the idea of ensemble learning, with decision trees and boosting as its core components.
+
+Ensemble learning is a technique in machine learning where multiple models, known as \textit{weak learners}, are combined to produce more accurate predictions.
+Mathematically, ensemble learning can be defined as combining the predictions of $M$ weak learners to form a final prediction $\hat{y}$, such that:
+\begin{equation}
+    \hat{y} = \sum_{m=1}^{M} \alpha_m \hat{y}_m
+\end{equation}
+where $\hat{y}_m$ is the prediction of the $m$-th weak learner and $\alpha_m$ is the weight assigned to the $m$-th weak learner.
+Although various options exist for weak learners, decision trees are commonly used and have been selected for this study.
+
+Decision trees partition the data into subsets based on feature values, aiming to create groups where the data points are more similar in their predicted outcomes. 
+This similarity, or homogeneity, is measured in terms of impurity, which is a measure of how mixed the data points are within a region. 
+This is achieved through recursive binary splitting, dividing the dataset into two regions based on the value of a predictor. 
+The goal at each step is to select the predictor and the split point that result in the greatest reduction in impurity.
+
+Impurity, in the context of decision trees, is measured using the residual sum of squares (RSS). For any predictor $X_j$ and split point $s$, the data is divided into two regions $R1$ and $R2$ defined as:
+$$
+\begin{aligned}
+    R1(j, s) &= \{X | X_j < s\} \\
+    R2(j, s) &= \{X | X_j \geq s\}
+\end{aligned}
+$$
+The goal is to find the values of $j$ and $s$ that minimize the RSS, calculated as:
+$$
+    \sum_{i: x_i \in R1(j, s)} (y_i - \hat{y}_{R1})^2 + \sum_{i: x_i \in R2(j, s)} (y_i - \hat{y}_{R2})^2
+$$
+where $\hat{y}_{R1}$ and $\hat{y}_{R2}$ are the mean responses for the training observations in regions $R1$ and $R2$, respectively.
```

Comment:
Impurity is just one of several metrics you can use to evaluate splits in a decision tree. There's also things like information gain. Why are you spending so much time explaining this specific evaluation metric in a section about gradient boosting? I don't think there's any reason to include that here. I would much rather we just say something like "...based on some evaluation metric", and perhaps list some examples. We are supposed to define gradient boosting in this section, not decision trees, and especially not model evaluation metrics.